docs/api-readme.md

```diff
@@ -0,0 +1,31 @@
+# LitElement API Documentation
+
+## Install @polymer/lit-element
+
+```
+npm install --save @polymer/lit-element@^1.0.0
```

Comment:
Sorry, my previous comment was "once we release 1.0" and just a note. We can't do this quite yet: https://jubianchi.github.io/semver-check/#/^1.0.0/1.0.0-rc.1